src/ucp/rndv/rndv.c

```diff
@@ -1297,6 +1282,16 @@ UCS_PROFILE_FUNC(ucs_status_t, ucp_rndv_rts_handler,
     }
 }
 
+static void ucp_rndv_ats_complete(ucp_request_t *sreq, ucs_status_t status)
```

Comment:
no need for separate func, which is called just once

Reply:
fixed